include/api.php

```diff
@@ -2518,7 +2518,7 @@ function api_get_entitities(&$text, $bbcode)
 		preg_match_all("/\[img](.*?)\[\/img\]/ism", $bbcode, $images);
 
 		foreach ($images[1] as $image) {
-			$replace = proxy_url($image);
+			$replace = Proxy::proxifyUrl($image);
```

Comment:
`proxifyUrl` should a method of `Util\Proxy`, while `Module\Proxy` should only be used to handle `/proxy/...` requests.

Reply:
Hmm, then only the namespace splits it. I would like to use `ProxyUtils` as class name.

Reply:
Not as a class name, but as a class alias like this: `use Friendica\Util\Proxy as ProxyUtil;`.

Reply:
Aliasing is what I find confusing. Let's name the class itself directly without aliasing it. I have seen it in Laravel and found it not clear code.

Reply:
You’ve seen aliasing or double-named classes?

Like I said, there’s no reason for both classes to be used alongside each on a regular basis, so the double-naming doesn’t seem warranted to me.

Reply:
Should I also move the constants over there?

Reply:
I think so, as they are not module-relevant. I avoided aliasing of classes now.

Reply:
Yes, `Proxy` (`Friendica\Module`) and `Proxy` (`Friendica\Util`) would be a double-naming. I try my best to avoid this to stop confusion.